odoo/addons/base/views/ir_translation_views.xml

```diff
@@ -33,7 +34,7 @@
                             <field name="lang"/>
                         </group>
                         <group>
-                            <field name="module" groups="base.group_no_one"/>
+                            <field name="module"/>
```

Comment:
well this view is no longer used anyway so we should probably get rid of it in the future...